src/eventhub.ts

```diff
@@ -467,22 +462,44 @@ class Eventhub extends EventEmitter implements IEventhub {
    * Delete key from key/value store.
    * @param key Key to delete.
    */
-  public del(key: string) : Promise<any> {
-    return this._sendRPCRequest(RPCMethods.DEL, { key: key });
+  public del(key: string): Promise<any> {
+    return this._sendRPCRequest(RPCMethods.DEL, { key });
```

Comment:
Eventhub expects { key: "keyName" }. This will only send { keyName } from my understanding. Or am I missing something here ? :)

Reply:
`{ key }` is a shorthand for `{ key: key }`
```
> key = 'keyName'
> { key }
{ key: 'keyName' }
```